CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   - Adjusted NodeClockNotSynchronising, NodeNetworkReceiveErrs, and NodeNetworkTransmitErrs alerts.
 - [#962](https://github.com/openshift/cluster-monitoring-operator/pull/962) Enable namespace by pod and pod total networking Grafana dashboards.
 - [#959](https://github.com/openshift/cluster-monitoring-operator/pull/959) Remove memory limits from prometheus-config-reloader in user workload monitoring
+- [#971](https://github.com/openshift/cluster-monitoring-operator/pull/971) Enable `hwmon` in node-exporter for hardware sensor data collection
```

Comment:
I remember us disabling not urgent collectors due to high cardinality, I am wondering how many new series this brings in on each cluster?

Reply:
The number of series depends on the number of sensors visible on the node, so it's hard to pin that down. On my dev system, I only get some temperature sensors. On other hosts, we should see more.

Reply:
@dhellmann I am wondering is this enablement part of some concrete initiative?

Reply:
Here's some example output from one host: https://paste.centos.org/view/e2d24856

That includes temperature data, but that host is apparently not exposing fan speed or other sensors.